client/js/libs/handlebars/tz.js

```diff
@@ -1,8 +1,10 @@
+var timestamp = window.timestamp;
+
 Handlebars.registerHelper(
 	"tz", function(time) {
 		if (time) {
-			var utc = moment.utc(time, "HH:mm:ss").toDate();
-			return moment(utc).format("HH:mm");
+			var utc = moment.utc(time, timestamp).toDate();
```

Comment:
Ah right, that explains why they were different. The server should probably send the whole datestamp, otherwise there are only hour, minutes and seconds possible, and this becomes pointless :-P

I'll take a look at sending the whole thing back from the server.